// Copyright 2013 <chaishushan{AT}gmail.com>. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package message;

message ArithRequest {
	optional int32 a = 1;
	optional int32 b = 2;
}

message ArithResponse {
	optional int32 c = 1;
}
